src/journal.c

```diff
@@ -103,7 +102,7 @@ static const UnkStruct_020E5B90 Unk_020E5B90[] = {
     { 0x46, 0x0 }
 };
 
-static const UnkStruct_020E5B50 Unk_020E5B50[] = {
+static const GymInfo sGymsInfo[] = {
     { 0xF6, 0x2F, 0x0 },
```

Comment:
**polish**: Tie these values to trainer constants (`res/trainers/trdata.naix`), map headers (`consts/maps.h`), and badges (`consts/badges.h`) while you're in this file.

Reply:
I've incorporated the maps and badges constants, but my branch doesn't have the trainers ones yet. I could either merge main into my branch and then do that as the last change, or I could do that for my next PR for the journal if that's also fine.